content/learn/getting-started/node-es6.fr.md

```diff
@@ -0,0 +1,39 @@
+---
+title: ecmascript-2015-es6-and-beyond
+displayTitle: 'ECMAScript 2015 (ES6) et plus'
+category: learn
+---
+
+Node.js est construit avec des versions modernes de [V8](https://v8.dev/). En restant à jour avec les dernières versions de ce moteur, nous nous assurons que les nouvelles fonctionnalités de la [spécification JavaScript ECMA-262](http://www.ecma-international.org/publications/standards/Ecma-262.htm) sont apportées aux développeurs Node.js en temps voulu, ainsi que des améliorations continues de la performance et de la stabilité.
```

Comment:
The link "[spécification JavaScript ECMA-262](http://www.ecma-international.org/publications/standards/Ecma-262.htm)" return to `https://www.ecma-international.org/publications-and-standards/standards/ecma-262/` (+SSL)

Reply:
Okay! I don't know if we can change that in French language to have directly the link. 
I suggested because it's the same link, it redirects to the HTTPS one and the final URL is slightly different. It avoids the inaccessible link later if there is a change on their side.

What do you think? 🤔

Reply:
by "convention" it would be necessary to open a pr with the change of link. But we can change it on the translation and open the pr at the same time.

Reply:
Okay np, I close this suggestion!

Reply:
> thanks for your review!

Np! I'm happy to help!